### PR TITLE
Flow utils file

### PR DESCRIPTION
**Pull request by chrisnojima**
Old:
```
/*::
      declare var ifFlowErrorsHereItsCauseYouDidntHandleAllTypesAbove: (a: empty) => any
      ifFlowErrorsHereItsCauseYouDidntHandleAllTypesAbove(action);
      */
```

New:
```
Flow.ifFlowComplainsAboutThisFunctionYouHaventHandledAllCasesInASwitch(action)
```

@keybase/react-hackers

### CODE REVIEW

File: shared/actions/chat2/index.js [hunk 1]
```diff
@@ -2527,26 +2525,24 @@ const loadStaticConfig = (state: TypedState, action: ConfigGen.DaemonHandshakePa
       })
     ),
     Saga.callUntyped(function*() {
-      const loadAction = yield RPCChatTypes.localGetStaticConfigRpcPromise().then(
-        (res: RPCChatTypes.StaticConfig) => {
```
Comment (zanderz) [inline shared/actions/chat2/index.js]: was this change intended for this PR?

Comment (chrisnojima) [reply shared/actions/chat2/index.js]: yea, we don't need to type res
